developer-docs-site/docs/tutorials/first-transaction.md

```diff
@@ -1,50 +1,90 @@
 ---
-title: "Your first transaction"
+title: "Your First Transaction"
 slug: "your-first-transaction"
 sidebar_position: 1
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Your first transaction
+# Your First Transaction
 
-This tutorial outlines how to generate, submit, and verify transactions submitted to the Aptos Blockchain. The steps for doing so are:
+This tutorial describes, in the following step-by-step approach, how to generate, submit, and verify transactions submitted to the Aptos Blockchain:
 
-1. Create a representation of an account
-2. Prepare a wrapper around the REST interfaces
-3. Prepare a wrapper around the Faucet interface
-4. Combine them into an application, execute and verify
+1. Create a representation of an account.
+  
+  Each Aptos account has a unique account address. The owner of that account holds the public, private key-pair that maps to the Aptos account address and, in turn, the authentication key stored in that account.  
+  
+  :::note
+  See more about Aptos accounts in [Accounts][account_basics]. 
+  :::
 
-The following tutorial contains example code that can be downloaded from our github below:
+2. Prepare a wrapper around the REST interfaces.
+
+  Aptos provides a [REST API][rest_spec] for interacting with the blockchain. This steps prepares wrappers around this API, for retrieving account information, and for constructing a transaction, signing it and submitting the transaction.
+
+3. Prepare a wrapper around the Faucet interface.
+
+  Using the Faucet interface at the Aptos devnet, this tutorial code automatically creates an account with the account address `0x1` and funds the account. 
+
+4. Combine the above wrappers into an application, execute and verify.
+
+## Before you start
+
+Make sure you follow the below steps first so you can run the tutorial. 
+
+1. Clone the Aptos repo.
```

Comment:
We should also mention that people should be checking out the devnet branch here. There might be some incompatibility problems if they don't do so.

Reply:
Good point.